README.md

```diff
@@ -52,6 +52,20 @@ abstract class MyPO {
 }
 ```
 
+**There are an addition two requirements for Dart files containing page objects:**
+1. Must be within either `/test/...` or `/testing/...` folder
```

Comment:
Why are we supporting 'testing'? Just curious.

Reply:
In case users want to keep their page objects separate from the actual executable tests. Think I should remove this requirement?

Reply:
Yes! Folks should put their test code in `test/src` or something. I'd remove this unless there is an explicit request...

Reply:
Sure, will do.